Bottom: 9e6f6d20209609d4abd22fe7096dfdb2624f579d
Top:    376d80de1f8cd65cd6dd9f22f6dd7be5b0d0df5b
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2013-01-03 02:29:45 +0000

configure.ac: Don't add GD_CFLAGS to CFLAGS, only CPPFLAGS

GD_CFLAGS will almost certainly contain pre-processor flags, not strictly
compiler flags, and we don't need to have a copy in CFLAGS as well.


---

diff --git a/configure.ac b/configure.ac
index eb6d238..63a2faa 100644
--- a/configure.ac
+++ b/configure.ac
@@ -792,7 +792,6 @@ you will get lucky.
 	# since some linux systems evidently install gdlib-config but fail to
 	# install the headers (nice), check for the header too and fail if it
 	# is not there.
-	CFLAGS="$CFLAGS $GD_CFLAGS"
 	CPPFLAGS="$CPPFLAGS $GD_CFLAGS"
 	AC_CHECK_HEADERS(gd.h)
 	case $ac_cv_header_gd_h in
